docs/source/llms/deployments/index.rst

```diff
@@ -858,7 +858,15 @@ Some providers support streaming responses. Streaming responses are useful when
 receive responses as they are generated, rather than waiting for the entire response to be
 generated before receiving it. Streaming responses are supported by the following providers:
 
-- OpenAI
++------------+---------------------+--------------+
+|  Provider  | Endpoints                          |
++------------+---------------------+--------------+
+|            | llm/v1/completions  | llm/v1/chat  |
++============+=====================+==============+
+| OpenAI     | ✓                   | ✓            |
++------------+---------------------+--------------+
+| Cohere     | ✓                   | ✘            |
```

Comment:
Cohere doesn't support chat?

https://docs.cohere.com/reference/chat

Reply:
We haven't implemented the non-streaming chat endpoint yet. And Cohere chat is in beta. Do you want me to implement both non-streaming & streaming chat?

Reply:
I think only the completions API existed when we added Cohere.

> Do you want me to implement both non-streaming & streaming chat?

Yes, if you have bandwidth to do that, but this PR can be merged without them.

Reply:
Ok, let's merge this without them. I'll create a new PR for chat